lib/rialto/etl/transformers/people/names.rb

```diff
@@ -53,6 +54,7 @@ def fullname_from_names(given_name:, middle_name: nil, family_name:)
           # @param family_name [String] last name
           # @return [Array(String)] array of name variations
           # rubocop:disable Metrics/MethodLength
+          # rubocop:disable Metrics/AbcSize
           def name_variations_from_names(given_name:, middle_name: nil, family_name:)
             name_variations = []
```

Comment:
Could this be Set.new rather than an array?

Reply:
Won't hurt.